part-i-getting-started/quick-start.md

```diff
@@ -0,0 +1,355 @@
+        remainingValue -= evt.getAmount();
+    }
+}
+```
+
+(1) `@AggregateIdentifier` annotation tells Axon that annotated field will be used as identifier of the Aggregate.
+
+(2) If you are using Axon for Event Sourcing, default constructor is needed so Axon can instantiate the Aggregate and apply all sourced events.
+
+> **Note** All business logic / rules are defined in the `@CommandHandler`s, and all state changes are defined in the `@EventSourcingHandler`s. The reason for this is when we want to get the current state of event-sourced Aggregate, we have to apply all sourced events - we have to invoke `@EventSourcingHandler`s. If the state of our Aggregate is changed outside of `@EventSourcingHandler`s it will not be reflected when we do a replay.
+
+### Query Model
+
+Once we have our Aggregate defined which processes commands and fires events, we can create a Query Model (Projection) based on those fired events. Let's say that we want to build a view that has summary of Giftcards. In order to do that, view will issue a query to our Query Model to retrieve necessary information about Giftcards. We will use good old `List` Java structure as our storage. `CardSummary` class could look like this:
+
+```java
+public class CardSummary {
+
+    private final String id;
+    private final Integer initialAmount;
+    private final Integer remainingAmount;
+
+    public CardSummary(String id, Integer initialAmount, Integer remainingAmount) {
+        this.id = id;
+        this.initialAmount = initialAmount;
+        this.remainingAmount = remainingAmount;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public Integer getInitialAmount() {
+        return initialAmount;
+    }
+
+    public Integer getRemainingAmount() {
+        return remainingAmount;
+    }
+
+    public CardSummary deductAmount(Integer toBeDeducted) {
+        return new CardSummary(id, initialAmount, remainingAmount - toBeDeducted);
+    }
+
+    @Override
+    public String toString() {
+        return "CardSummary{" +
+                "id='" + id + '\'' +
+                ", initialAmount=" + initialAmount +
+                ", remainingAmount=" + remainingAmount +
+                '}';
+    }
+```
+
+Before building a Query Model, we should define a query which would retrieve card summaries with predefined size and offset:
+
+```java
+public class FetchCardSummariesQuery {
+
+    private final Integer size;
+    private final Integer offset;
+
+    public FetchCardSummariesQuery(Integer size, Integer offset) {
+        this.size = size;
+        this.offset = offset;
+    }
+
+    public Integer getSize() {
+        return size;
+    }
+
+    public Integer getOffset() {
+        return offset;
+    }
+}
+```
+
+Query is, as commands and events, a POJO. We can start building our Query Model.
+
+```java
+public class CardSummaryProjection {
+
+    private final List<CardSummary> cardSummaries = new CopyOnWriteArrayList<>();
+
+    @EventHandler
```

Comment:
I kinda liked the comments you added as footnotes, and I think it's valuable for the `@EventHandler` annotation as well.